examples/v0/account_management/create_customer.py

```diff
@@ -32,11 +32,14 @@ def main(client, customer_id):
     resource_name = customer_service.customer_path(customer_id)
```

Comment:
It looks like indentation is wrong starting here through the try block (interestingly... the except block is O.K.)

Reply:
Is this not indented by 4 spaces? It looks correct to me.

Reply:
I'm seeing an indent of six spaces over here for `resource_name`. Is that not the case?

Reply:
I think that's just formatting applied by GitHub. If you click "view file" above it'll display the unformatted code and you should only see 4 spaces.